CHANGELOG.md

```diff
@@ -1,30 +1,30 @@
 # Release History
 
-## 1.4.0 (TBD)
+## 1.4.0 (2023-04-20)
 
 ### New Features
 
-- Added support for `session.getOrCreate`
-- Added support for `functions.array_distinct`
-- Added support for alias `Column.getField`
-- Added support for `functions.regexp_extract`
-- Added support for `functions.struct`
-- Added support for `functions.daydiff` which is compatible with spark datediff
-- Added support for `functions.date_add` and `functions.date_sub` to ease common day add/substract operations
-- Added support for `functions.format_number`
-- Added support for `functions.bround`
+- Added support for `session.getOrCreate`.
+- Added support for `functions.array_distinct`.
+- Added support for alias `Column.getField`.
+- Added support for `functions.regexp_extract`.
+- Added support for `functions.struct`.
+- Added support for `functions.daydiff` which is compatible with Spark datediff.
+- Added support for `functions.date_add` and `functions.date_sub` to make add and subtract operations easier.
+- Added support for `functions.format_number`.
```

Comment:
No strong opinion but I wonder if we should group the newly introduced functions under one bullet point and add sub-bullets for ones that needs explanation. For example:

```
- Added support for new functions:
  - `functions.daydiff` which is compatible with Spark datediff.
  - `functions.date_add` and `functions.date_sub` to make add and subtract operations easier.
  - Other functions
```

Reply:
I'm working on it